Tools/SandboxTest.ps1

```diff
@@ -1,269 +1,518 @@
-# Parse arguments
-[Diagnostics.CodeAnalysis.SuppressMessageAttribute('PSAvoidUsingWriteHost', '', Justification = 'This script is not intended to have any outputs piped')]
-[Diagnostics.CodeAnalysis.SuppressMessageAttribute('PSReviewUnusedParameter', 'Prerelease', Justification = 'The variable is used in a conditional but ScriptAnalyser does not recognize the scope')]
-[Diagnostics.CodeAnalysis.SuppressMessageAttribute('PSReviewUnusedParameter', 'WinGetVersion', Justification = 'The variable is used in a conditional but ScriptAnalyser does not recognize the scope')]
-
+﻿### Exit Codes:
+# -1 = Sandbox is not enabled
+#  0 = Success
+#  1 = Error fetching GitHub release
+#  2 = Unable to kill a running process
+#  3 = WinGet is not installed
+#  4 = Manifest validation error
+###
+
+[CmdletBinding()]
 Param(
-  [Parameter(Position = 0, HelpMessage = 'The Manifest to install in the Sandbox.')]
-  [String] $Manifest,
-  [Parameter(Position = 1, HelpMessage = 'The script to run in the Sandbox.')]
-  [ScriptBlock] $Script,
-  [Parameter(HelpMessage = 'The folder to map in the Sandbox.')]
-  [String] $MapFolder = $pwd,
-  [switch] $SkipManifestValidation,
-  [switch] $Prerelease,
-  [switch] $EnableExperimentalFeatures,
-  [string] $WinGetVersion,
-  [Parameter(HelpMessage = 'Additional options for WinGet')]
-  [string] $WinGetOptions
+    # Manifest
+    [Parameter(Position = 0, HelpMessage = 'The Manifest to install in the Sandbox.')]
+    [ValidateScript({
+            if (-Not (Test-Path -Path $_)) { throw "$_ does not exist" }
+            return $true
+        })]
+    [String] $Manifest,
+    # Script
+    [Parameter(Position = 1, HelpMessage = 'The script to run in the Sandbox.')]
+    [ScriptBlock] $Script,
+    # MapFolder
+    [Parameter(HelpMessage = 'The folder to map in the Sandbox.')]
+    [ValidateScript({
+            if (-Not (Test-Path -Path $_ -PathType Container)) { throw "$_ is not a folder." }
+            return $true
+        })]
+    [String] $MapFolder = $pwd,
```

Comment:
If I launch the script from `C:\`, something like `& C:\Users\<user>\winget-pkgs\Tools\SandboxTest.ps1`, this will give sandbox read & write access to my entire system, which may not be obvious to the user running the script, and can allow an app inside sandbox to do anything it wants.

Should we disallow calling the script this way, or warn user when a parent path of `winget-pkgs` folder is marked as read-or-write?

Reply:
I realize this is not strictly relevant to the refactor changes here as it was the same behavior before, but I thought it was worth calling out

Reply:
I like the idea of disallowing it, but that could cause issues in some use cases. Take for example my setup - I have everything inside of a Dev drive. It could be possible that I want to open the sandbox with read-write into the dev drive. 

I do think that adding a warning if the Map-Folder doesn't contain `winget-pkgs` would be a good idea though